rotkehlchen/globaldb/updates.py

```diff
@@ -300,74 +306,230 @@ def _parse_full_insert(self, insert_text: str) -> AssetData:
+                        f'Error: {str(e)}',
+                    )
+                return  # fail or succeed continue to next entry
+
+            if local_asset is not None:
+                AssetResolver().clean_memory_cache(local_asset.identifier.lower())
+
+            # otherwise asset is known, so it's a conflict. Check if we can resolve
+            resolution = assets_conflicts.get(local_asset) if assets_conflicts else None
+            if resolution == 'local':
+                # do nothing, keep local
+                return
+            if resolution == 'remote':
+                try:
+                    with connection.savepoint_ctx() as cursor:
+                        _force_remote_asset(cursor, local_asset, full_insert)
+                except sqlite3.Error as e:
+                    self.msg_aggregator.add_warning(
+                        f'Failed to resolve conflict for {remote_asset_data.identifier} in '
+                        f'the DB during the v{version} assets update. Skipping entry. '
+                        f'Error: {str(e)}',
+                    )
+                return  # fail or succeed continue to next entry
+
+            # else can't resolve. Mark it for the user to resolve.
+            # TODO: When assets refactor is finished, remove the usage of AssetData here
+            local_data = GlobalDBHandler().get_all_asset_data(  # pylint: disable=unsubscriptable-object  # noqa: E501
+                mapping=False,
+                serialized=False,
+                specific_ids=[local_asset.identifier],
+            )[0]
+            self.conflicts.append((local_data, remote_asset_data))
+
+    def _apply_single_version_update(
+            self,
+            connection: 'DBConnection',
+            version: int,
+            text: str,
+            assets_conflicts: Optional[dict[Asset, Literal['remote', 'local']]],
+    ) -> None:
+        """
+        Process each action in the queried data and process it by:
+        1. Try to deserialize it as an asset
+        2. Try to deserialize it as an asset collection
+        3. Try to deserialize it as an asset collection mapping.
+
+        If none of the above rules sucess then the entry is skipped. In the case of assets
+        possible conflicts are handled.
+        """
+        lines = text.splitlines()
+        for action, full_insert in zip(*[iter(lines)] * 2):
+            if full_insert.strip() == '*':
+                full_insert = action
+
+            if (
+                (match := self.assets_re.match(full_insert)) is not None and
+                len(match.groups()) != 0
+            ):
+                remote_asset_data = None
+                try:
+                    remote_asset_data = self._parse_full_insert_assets(full_insert)
+                except DeserializationError as e:
+                    self.msg_aggregator.add_warning(
+                        f'Skipping entry during assets update to v{version} due '
```

Comment:
Wouldn't it make sense to add a log entry with the raw entry we skip? Since by seeing this in the logs we won't be able to know which entry it's talking about

Reply:
you still have not done this